example/spec/syncGranule/SyncGranuleDuplicateHandlingVersionSpec.js

```diff
@@ -0,0 +1,224 @@
+const fs = require('fs');
+const path = require('path');
+const {
+  aws: {
+    headObject,
+    parseS3Uri,
+    s3
+  },
+  stringUtils: { globalReplace },
+  testUtils: {
+    randomString
+  }
+} = require('@cumulus/common');
+const { api: apiTestUtils, buildAndExecuteWorkflow, LambdaStep } = require('@cumulus/integration-tests');
+const {
+  deleteFolder,
+  loadConfig,
+  templateFile,
+  timestampedTestDataPrefix,
+  uploadTestDataToBucket
+} = require('../helpers/testUtils');
+const {
+  loadFileWithUpdatedGranuleId,
+  setupTestGranuleForIngest
+} = require('../helpers/granuleUtils');
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const workflowName = 'SyncGranuleDuplicateVersionTest';
+
+const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$';
+const testDataGranuleId = 'MOD09GQ.A2016358.h13v04.006.2016360104606';
+
+const outputPayloadTemplateFilename = './spec/syncGranule/SyncGranule.output.payload.template.json';
+const templatedOutputPayloadFilename = templateFile({
+  inputTemplateFilename: outputPayloadTemplateFilename,
+  config: config.SyncGranule
+});
+
+const s3data = [
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606_ndvi.jpg'
+];
+
+describe('When the Sync Granule workflow is configured to keep both files when encountering duplicate filenames', () => {
+  const testDataFolder = timestampedTestDataPrefix(`${config.stackName}-IngestGranuleSuccess`);
```

Comment:
Update this string to the name of your spec, e.g. SyncGranuleDuplicateHandlingVersion